.travis.yml

```diff
@@ -17,6 +19,7 @@ matrix:
       node_js: 10
       before_install:
         - npm install request
+        - npm install --package-lock-only
```

Comment:
I've been thinking we should move to `npm ci`, which I think does this. Do you know if there is a functional or important difference between `npm ci` and `npm install --package-lock-only`?

Reply:
actually, the command that is executed further down is [npm ci](https://github.com/mapbox/node-pre-gyp/pull/540/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R53) and that fails because package.json and package-lock.json are out of sync. at least that's what i was seeing. that's why only this test was failing - it's the only one that did `npm install request`